setup.py

```diff
@@ -87,6 +96,28 @@ def get_hipcc_rocm_version():
         return None
 
 
+def get_neuronxcc_version():
+    # Run the neuronx-cc --version command
+    result = subprocess.run(['neuronx-cc', '--version'],
+                            stdout=subprocess.PIPE,
+                            stderr=subprocess.STDOUT,
+                            text=True)
+
+    # Check if the command was executed successfully
+    if result.returncode != 0:
+        print("Error running 'neuronx-cc --version'")
```

Comment:
Why don't we just raise an error in this case?

Reply:
I was trying to be consistent with how we handle `hipcc --version` failure.

https://github.com/vllm-project/vllm/blob/a1b9cb2a3469e4e682e741af9a0f91e16923205d/setup.py#L77

Reply:
raising a RuntimeError